packages/components/psammead-headings/README.md

```diff
@@ -47,13 +51,15 @@ This usage will allow for the page anchor: `www.bbc.com/news/articles/articleID#
 
 ### When to use this component
 
-These components can be used at any point on the page, however the `Headline` is designed to be used once at the top of the page. The `SubHeading` takes an optional `id` value and passes it to the `h2` which can be used as an anchor when referencing content.
+These components can be used at any point on the page, however the `Headline` and `PageHeading` are designed to be used once at the top of the page.
+
+The `SubHeading` and `PageHeading` can take an optional `id` attribute which can be used as an anchor when referencing content.
 
 <!-- ### When not to use this component -->
 
 ### Accessibility notes
 
-The `SubHeading` component has a tabindex of `-1` so that it works correctly with screen readers when navigated to via a skip link.
+The `SubHeading` and `PageHeading` component have a tabindex of `-1` so that they work correctly with screen readers when navigated to via a skip link.
```

Comment:
Please can we remove 'skip link' - This functionality for subheads wasn't designed for skip links specifially.

Reply:
I no you haven't added that....

Reply:
How about? "The `SubHeading` and `PageHeading` component have a tabindex of `-1` this ensures that these elements are focusable by assitive technology."